src/Core/Framework/Plugin/PluginLifecycleService.php

```diff
@@ -113,12 +117,13 @@ public function installPlugin(PluginEntity $plugin, Context $shopwareContext): I
         $pluginBaseClass = $this->getPluginBaseClass($plugin->getBaseClass());
         $pluginVersion = $plugin->getVersion();
 
+        $migrations = $this->createMigrationCollection($pluginBaseClass);
```

Comment:
Can you add a testcase for this change?

You can add it to `\Shopware\Core\Framework\Test\Plugin\PluginLifecycleServiceTest`, there are also fixture plugins, please add a plugin with own migrations and migrations in a bundle and check that both migrations are executed when the plugin is installed or updated.

And while your own it, please add a small changelog.

Reply:
I can do that :) thanks for the clear instruction, where to dive into next.